src/models/activity_result.js

```diff
@@ -3,6 +3,18 @@ import State from 'ampersand-state';
 export default State.extend({
   extraProperties: 'allow',
 
+  resultType: 'url',
+  derived: {
```

Comment:
If we're adding `derived` (which I just learned existed; neat!), could you move `displayTitle` and `displayImage` in there too?